packages/react-emotion/types/index.d.ts

```diff
@@ -4,6 +4,7 @@ import { StatelessComponent, ComponentClass, CSSProperties } from 'react';
 import { Interpolation as EmotionInterpolation } from 'emotion';
 
 export * from 'emotion';
+export * from 'emotion-theming';
```

Comment:
This should be updated to match @mitchellhamilton's work in #457

Reply:
I don't understand, #457 shouldn't require any changes to this PR.

Reply:
Oops, I thought you manually changed the export in that PR. Didn't realize the update fixed it.